### PR TITLE
Add py3.6, 3.8, 3.9, Docker support to PrivacyRaven

### PR DESCRIPTION
**Pull request by pwang00**
These changes should address #52 and allow PrivacyRaven to be used on Google Colab.

### CODE REVIEW

File: examples/create_synthesizer.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 
 # Create a query function for a target PyTorch Lightning model
-model = train_four_layer_mnist_victim(gpus=1)
+model = train_four_layer_mnist_victim(gpus=torch.cuda.device_count())
```
Comment (suhacker1) [inline examples/create_synthesizer.py:12]: Add a comment explaining how this is different from manually setting the gpus argument to 1 or 0.

File: examples/extract_mnist_cpu.py [hunk 2]
```diff
@@ -12,6 +12,16 @@
 from privacyraven.models.victim import train_four_layer_mnist_victim
 from privacyraven.models.four_layer import FourLayerClassifier
 
+
+# PrivacyRaven's builtin victim functions allow the user to specify the number of GPUs
+# to use during computation via the `gpus` keyword argument.  By default, this is 1, but 
+# setting gpus=torch.cuda.device_count() will let PrivacyRaven / Pytorch use all available GPUs.
```
Comment (suhacker1) [inline examples/extract_mnist_cpu.py:18]: I think putting this comment here is too much information for one of the starter examples.

File: examples/extract_mnist_gpu.py [hunk 3]
```diff
@@ -11,10 +11,19 @@
 from privacyraven.models.victim import train_four_layer_mnist_victim
 from privacyraven.models.four_layer import FourLayerClassifier
 
-# Create a query function for a target PyTorch Lightning model
-model = train_four_layer_mnist_victim(gpus=1)
 
+# PrivacyRaven's builtin victim functions allow the user to specify the number of GPUs
+# to use during computation via the `gpus` keyword argument.  By default, this is 1, but 
+# setting gpus=torch.cuda.device_count() will let PrivacyRaven / Pytorch use all available GPUs.
```
Comment (suhacker1) [inline examples/extract_mnist_gpu.py:17]: I think putting this comment here is too much information for one of the starter examples.

Comment (pwang00) [reply examples/extract_mnist_gpu.py:17]: Do you want me to move this comment or just condense it?

**Comment by suhacker1** [comment]
Everything looks good!

**Pull request merged**